apps/emqx_bridge_dynamo/src/emqx_bridge_dynamo_connector.erl

```diff
@@ -291,6 +298,22 @@ do_query(
             Result
     end.
 
+trace_render_result(RenderResult, #{trace_ctx := LogMetaData, action_id := ActionID}) ->
```

Comment:
identical implementation for tdengin, mqtt, rabbitmq.
extract to `emqx_trace.erl`

```
make_template_render_tracer_fn(Context) ->
    {fun ?MODULE:trace_render_result/2, Context}.

eval_teamplate_render_tracer_fn({Fn, Context}) ->
    ...
```

Reply:
Good suggestion. Will fix.